netket/operator/_pauli_strings.py

```diff
@@ -493,6 +487,19 @@ def gccf_fun(x, sections):
         return jit(nopython=True)(gccf_fun)
 
 
+def _count_of_locations(of_qubit_operator):
+    # we always start counting from 0, so we only determine the maximum location
+    def max_or_default(x):
+        x = list(x)
```

Comment:
I think you can drop this, unless openfermion uses a weird storage format?
```suggestion
```

Reply:
ah, this is just to make a list from the generator to be able to call len(x)

Reply:
ah right. I thought you could call len on it directly. Nevermind then